README.md

```diff
@@ -135,7 +147,9 @@ The `symengine` dependency is built from a local conan recipe. Run:
 conan create --profile=tket recipes/symengine
 ```
 
-to build it.
+to build it. If you are using a conan configuration supported by the CI, this is
```

Comment:
From my understanding this only works for specific compiler versions? If that is a case should we add them here?

Reply:
Yes that's right. There is a list above under "Build tools". Do you think that is enough? Maybe we should add a reference to that section here.

Reply:
I have seen you added that, thank you!